staging/src/k8s.io/apiserver/pkg/server/signal.go

```diff
@@ -44,6 +45,26 @@ func SetupSignalHandler() <-chan struct{} {
 	return stop
 }
 
+// SetupSignalContext is same as SetupSignalHandler, but a context.Context is returned.
```

Comment:
the definition of these two methods will drift... can we change SetupSignalHandler to:

```go
func SetupSignalHandler() <-chan struct{} {
  return SetupSignalContext().Done()
}
```

Reply:
sure, already done